.github/workflows/release-deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: release-deploy
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [ '3.10' ]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: main
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build package
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install wheel
+        cd main
+        echo "RELEASE_VERSION=$(cat pyfakefs/_version.py | cut -d "'" -f 2)" >> $GITHUB_ENV
+        python setup.py sdist bdist_wheel
+
+    - name: Publish package to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
If it is all the same to you, I suggest using `PYPI_TOKEN` for consistency with other repositories in pytest-dev (but this is not 100% consistent either, so feel free to disregard this).

Reply:
Sure, will change this.